static/app/components/charts/components/tooltip.tsx

```diff
@@ -402,3 +402,6 @@ export default function Tooltip({
     ...props,
   };
 }
+
+const DoNotUseChartTooltip = ChartTooltip;
+export default DoNotUseChartTooltip;
```

Comment:
are we planning to replace default imports in a follow-up? it would be nice to not have a default here anymore